include/swift/Option/Options.td

```diff
@@ -1326,6 +1326,11 @@ def pretty_print: Flag<["-"], "pretty-print">,
   Flags<[SwiftAPIExtractOption, SwiftSymbolGraphExtractOption]>,
   HelpText<"Pretty-print the output JSON">;
 
+def emit_extension_block_symbols: Flag<["-"], "emit-extension-block-symbols">,
+  Flags<[SwiftSymbolGraphExtractOption, FrontendOption,
+        NoInteractiveOption, SupplementaryOutput, HelpHidden]>,
+  HelpText<"Emit 'swift.extension' symbols for extensions to external types instead of directly associating members and conformances with the extended nominal">;
+
```

Comment:
Ideally we would want to make this behaviour the default so usability of the flag is not a major concern, so it might be worth indicating in the flag name itself that this is a symbol graph extraction only feature. Maybe `-emit-symbol-graph-with-extension-block-symbols`. At least the help text should definitely mention symbol graphs otherwise it isn't very clear that this is a symbol graph feature.

Reply:
Good catch, thank you...I guess I got a little lost in my own world there!

Reply:
I think it needs to be more explicit than this.